yarn.lock

```diff
@@ -6442,6 +6447,11 @@
   resolved "https://registry.npmjs.org/@uniswap/v3-core/-/v3-core-1.0.0.tgz"
```

Comment:
why is 1.0.0 still required? this is a duplicate that needs to be resolved.

Reply:
Several dependencies still rely on it include the latest of `router-sdk`

Reply:
this is forcing us to ship two copies of v3-core. Can we fix it upstream?

Reply:
https://linear.app/uniswap/issue/WEB-2515/no-longer-import-uniswapv3-core-v100